packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -948,6 +949,10 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
       return -1;
     }
 
+    if (typeof selectedKey === 'number') {
+      return selectedKey;
+    }
+
```

Comment:
This changes the logic of getSelectedIndex which will likely break something that relies on this. It's not clear to me why this is needed. You should know the selectedKey for the item you want, you can't assume that if the selectedKey is a number that it refers to the selected index.

Reply:
@jspurlin Yes! this isn't needed at all.  It has been removed.